x-pack/plugins/dataset_quality/server/routes/data_streams/get_data_stream_details/index.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ElasticsearchClient } from '@kbn/core/server';
+import { DataStreamTypes } from '../../../types/default_api_types';
+import { DataStreamDetails } from '../../../../common/api_types';
+import { dataStreamService } from '../../../services';
+
+export async function getDataStreamDetails(args: {
+  esClient: ElasticsearchClient;
+  type: DataStreamTypes;
+  datasetQuery: string;
```

Comment:
Could we rename this to something like `dataset` or `index`? If we follow the assumption that we will query directly the required index then having this as a potential query could be confusing

Reply:
Renamed to `dataStream`.